packages/devextreme/testing/testcafe/helpers/disposeWidget.ts

```diff
@@ -0,0 +1,27 @@
+import { ClientFunction } from 'testcafe';
+import { WidgetName } from './widgetName';
+
+export default async function disposeWidget<TWidgetName extends WidgetName>(
+  componentName: TWidgetName,
```

Comment:
minor
no need for generic since nothing depends on this type
```suggestion
export default async function disposeWidget(
  componentName: WidgetName,
```